docs/tutorials/uky_flowsheet-solution.ipynb

```diff
@@ -1991,48 +1991,61 @@
       "        inequality constraints with only lower bounds:        0\n",
       "   inequality constraints with lower and upper bounds:        0\n",
       "        inequality constraints with only upper bounds:        0\n",
       "\n",
       "iter    objective    inf_pr   inf_du lg(mu)  ||d||  lg(rg) alpha_du alpha_pr  ls\n",
-      "   0  0.0000000e+00 4.04e+05 1.00e+00  -1.0 0.00e+00    -  0.00e+00 0.00e+00   0\n",
-      "   1  0.0000000e+00 1.51e+05 3.59e+07  -1.0 8.34e+05    -  9.21e-05 6.28e-01h  1\n",
-      "   2  0.0000000e+00 8.44e+03 4.43e+06  -1.0 1.31e+05    -  2.29e-02 9.44e-01h  1\n",
-      "   3  0.0000000e+00 9.23e+01 3.69e+05  -1.0 2.64e+04    -  7.32e-01 9.90e-01h  1\n",
-      "   4  0.0000000e+00 2.40e+00 2.75e+03  -1.0 3.27e+02    -  9.86e-01 9.91e-01h  1\n",
-      "   5  0.0000000e+00 3.61e-03 3.50e+04  -1.0 2.90e+00    -  9.90e-01 1.00e+00h  1\n",
-      "   6  0.0000000e+00 2.33e-10 8.73e+04  -1.0 1.09e-06    -  9.90e-01 1.00e+00h  1\n",
+      "   0  0.0000000e+00 2.50e+05 1.00e+00  -1.0 0.00e+00    -  0.00e+00 0.00e+00   0\n",
+      "   1  0.0000000e+00 9.29e+04 3.59e+07  -1.0 8.34e+05    -  9.21e-05 6.28e-01h  1\n",
+      "   2  0.0000000e+00 5.21e+03 4.43e+06  -1.0 1.31e+05    -  2.29e-02 9.44e-01h  1\n",
+      "   3  0.0000000e+00 6.06e+01 3.69e+05  -1.0 2.64e+04    -  5.51e-01 9.90e-01h  1\n",
+      "   4  0.0000000e+00 8.56e+00 2.23e+04  -1.0 3.27e+02    -  5.71e-01 9.91e-01h  1\n",
+      "   5  0.0000000e+00 1.29e+00 1.49e+06  -1.0 6.09e+00    -  5.77e-01 1.00e+00h  1\n",
+      "   6  0.0000000e+00 1.80e-06 3.83e+06  -1.0 3.91e+00    -  5.84e-01 1.00e+00f  1\n",
+      "   7  0.0000000e+00 2.71e-05 1.54e+06  -1.0 9.19e+00    -  5.99e-01 1.00e+00f  1\n",
+      "   8  0.0000000e+00 2.35e-04 7.48e+05  -1.0 2.52e+01    -  5.13e-01 1.00e+00f  1\n",
+      "   9  0.0000000e+00 1.34e-03 3.68e+05  -1.0 6.67e+01    -  5.08e-01 1.00e+00f  1\n",
+      "iter    objective    inf_pr   inf_du lg(mu)  ||d||  lg(rg) alpha_du alpha_pr  ls\n",
+      "  10  0.0000000e+00 5.79e-04 3.68e+03  -1.0 3.55e+01  -4.0 9.90e-01 1.00e+00f  1\n",
+      "  11  0.0000000e+00 7.15e-04 1.72e+06  -1.7 5.54e+01    -  7.85e-01 1.00e+00f  1\n",
+      "  12  0.0000000e+00 3.64e-04 1.71e+04  -1.7 3.69e+01  -4.5 9.90e-01 1.00e+00f  1\n",
+      "  13  0.0000000e+00 4.74e-08 1.46e+04  -3.8 1.10e+00    -  9.93e-01 1.00e+00h  1\n",
+      "  14  0.0000000e+00 4.44e-03 2.09e+03  -3.8 1.26e+02    -  8.57e-01 1.00e+00f  1\n",
+      "  15  0.0000000e+00 4.77e-02 1.37e+03  -3.8 7.76e+02    -  3.42e-01 1.00e+00h  1\n",
+      "  16  0.0000000e+00 1.36e-04 4.11e-04  -3.8 3.70e+01  -5.0 1.00e+00 1.00e+00h  1\n",
+      "  17  0.0000000e+00 5.39e-10 1.41e-05  -5.7 1.13e-01  -5.4 1.00e+00 1.00e+00h  1\n",
+      "  18  0.0000000e+00 2.33e-10 3.96e-08  -8.6 5.59e-05  -5.9 1.00e+00 1.00e+00h  1\n",
       "\n",
-      "Number of Iterations....: 6\n",
+      "Number of Iterations....: 18\n",
```

Comment:
Why did the number of iterations increase so much?

Reply:
Thanks for catching this. I didn't carry over all of the necessary changes to this tutorial. Now it still solves in 6 iterations